singer-io/tap-mambu
# v.1.0.4 Adjust paginationDetails

# Description of change
Remove `paginationDetails` parameter from select paged API calls; no longer needed after recent change to Mambu API. This change requested by Mambu to improve Mambu API server performance.

# Manual QA steps
Changed endpoints requested by Mambu. Ran singer-discover, singer-check-tap, and target-stitch (initial load and incremental load). Verified data in downstream target DB destination. No errors.
 
# Risks
Low - minor change to parameters for 6 endpoints.
 
# Rollback steps
Revert to v.1.0.3
